FV3/coarse_graining/coarse_graining.F90

```diff
@@ -160,7 +171,10 @@ subroutine assert_valid_strategy(strategy)
 
     character(len=256) :: error_message
 
-    if (trim(strategy) .ne. MODEL_LEVEL .and. trim(strategy) .ne. PRESSURE_LEVEL) then
+    if (trim(strategy) .ne. MODEL_LEVEL .and. &
```

Comment:
It could be worth splitting this into `MODEL_LEVEL_AREA_WEIGHTED` and `MODEL_LEVEL_MASS_WEIGHTED` options as well  Currently this implicitly means `MODEL_LEVEL_MASS_WEIGHTED`.  I might do that in another PR.